console/console-init/ui/src/modules/iot-project-detail/DetailPage.tsx

```diff
@@ -0,0 +1,76 @@
+import React from "react";
+import { Grid, GridItem } from "@patternfly/react-core";
+import {
+  GeneralInfo,
+  DeviceRegistationManagement,
+  AccessCredentials,
+  IMessagingObject
+} from ".";
+import { action } from "@storybook/addon-actions";
+import { IAdapterConfig, IAdapter } from "components/AdapterList";
+import { Protocols } from "constant";
+
+export default function DetailPage() {
```

Comment:
please rename DetailPage to some thing meaning ful like IoTProjectDetailInfoPage or some thing else